static/app/components/feedback/list/feedbackListItem.tsx

```diff
@@ -95,6 +96,11 @@ const FeedbackListItem = forwardRef<HTMLDivElement, Props>(
           <div style={{gridArea: 'message'}}>
             <TextOverflow>{feedbackItem.metadata.message}</TextOverflow>
           </div>
+          <div style={{gridArea: 'assigned', display: 'flex', justifyContent: 'end'}}>
```

Comment:
Can we use the `<Flex>` here?

Reply:
i believe that component is `AssigneeSelector` which requires passing in some sort of `MemberList` and i wasn't sure what goes there. and yes we can! converted this to draft for now since we won't be publishing the assignee dropdown yet. doesn't make sense to display this yet either